pkgs/os-specific/linux/kernel/linux-beagleboard.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, hostPlatform, fetchFromGitHub, perl, buildLinux, ... } @ args:
+
+let
+  modDirVersion = "4.9.59";
+  tag = "r73";
+in
+import ./generic.nix (args // rec {
+  version = "${modDirVersion}-ti-${tag}";
+  inherit modDirVersion;
+
+  src = fetchFromGitHub {
+    owner = "beagleboard";
+    repo = "linux";
+    rev = "${version}";
+    sha256 = "1kzbbaqmzgvfls1v9jir2ck9vcdd774mq474vhr5x6dqjnnb5kg9";
+  };
+
+  kernelPatches = args.kernelPatches;
+
+  features.iwlwifi = true;
```

Comment:
I think this `features` section needs to be:
````
  features = {
    efiBootStub = false;
  } // (args.features or {});
````
if `CONFIG_EFI_STUB` is not enabled, or dropped if not (since the default is 'true'). This was changed in commit 8525b78d054ac7ef2c43057d54403846b2d07ccc

Reply:
Thanks. I wrote this several months ago and didn't catch the change.